packages/terminal/src/browser/base/terminal-service.ts

```diff
@@ -15,10 +15,12 @@
  ********************************************************************************/
 import { Event } from '@theia/core/lib/common/event';
 import { WidgetOpenerOptions } from '@theia/core/lib/browser';
-import { TerminalWidgetOptions, TerminalWidget } from './terminal-widget';
+import { TerminalWidgetOptions, TerminalWidget } from '../terminal-widget-impl';
 
 /**
  * Service manipulating terminal widgets.
+ * @since 1.1.0
```

Comment:
please remove `@since`

- it states bogus information, `TerminalService` was introduced much earlier than 1.1.0
- it should be used only in couple with explicit stability tag: https://github.com/eclipse-theia/theia/blob/master/doc/api-management.md#stability